airbyte/manifest.json

```
@@ -0,0 +1,37 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "5994a02c-8754-40c3-9e99-a39ffc862b1c",
+  "app_id": "airbyte",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor the state of your Airbyte deployment.",
+    "title": "Airbyte",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
```

Would any additional [classifier tags](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags) apply here?  Feel free to discard this suggestion if it doesn't apply.

```suggestion
      "Category::Data Store",
      "Supported OS::Linux",
```

---

Not really since the Python package part of this integration is a placeholder.